assets/js/signup.js

```diff
@@ -23,7 +11,7 @@ var loadQuestions = function() {
   });
 
   $.getJSON({
-    url: '/server/form.php?retrieve=questions',
+    url: '/server/form.php?retrieve=questions&subheadings[]=Contact Information&subheadings[]=Language Information&subheadings[]=Background Information',
```

Comment:
Yeah. We don't currently have a method of imposing an ordering on the subheadings. That might be something that we would need a new table for. I did this for now so that the specific subheadings would not (even though they are currently hard-coded) be sitting in the form.php file. The ordering before now was alphabetic by subheading, but that isn't a very practical or useful solution for this.